rest/route/admin_routes.go

```diff
@@ -15,7 +15,59 @@ import (
 	"github.com/pkg/errors"
 )
 
-// this manages the /admin route, which allows getting/setting the admin settings
+// this route is to preserve backwards compatibility with old versions of the CLI
+func getLegacyAdminSettingsManager(route string, version int) *RouteManager {
+	agh := &legacyAdminGetHandler{}
+	adminGet := MethodHandler{
+		PrefetchFunctions: []PrefetchFunc{PrefetchUser},
+		Authenticator:     &RequireUserAuthenticator{},
```

Comment:
Since you can see the banner if you're not logged in, this should not have auth.